docs/connector-development/connector-builder-ui/overview.md

```diff
@@ -1,57 +1,48 @@
 
-- You want to integrate with a JSON-based HTTP API as a source of records
-- The API you want to integrate with doesn't exist yet as a connector in the [connector catalog](/integrations/sources/).
-- The API is suitable for the connector builder as per the
-  [compatibility guide](./connector-builder-compatibility.md).
+**Configured Source**: The configured source is what you set up when you provide the variables needed for the connector to access records. The exact fields of the configuration depend on the connector, but in most cases, it provides authentication information (username and password, API key) and information about which data to extract, for example, the start date to sync records from, a search query records have to match.
 
-## Getting started
+**Connection**: A connection is an automated data pipeline that replicates data from a source to a destination. It links a configured source (based on a source connector) to a configured destination (based on a destination connector) to perform syncs. It defines things like the replication frequency (e.g. hourly, daily, manually) and which streams to replicate.
 
-The high level flow for using the connector builder is as follows:
-
-1. Access the connector builder in the Airbyte web app
-2. Use the connector builder to iterate on your low-code connector
-3. Once the connector is ready, publish it to the local workspace
-4. Configure a Source based on the released connector
-5. Use the Source in a connection to sync data
+## When should I use the connector builder?
 
-Follow [the tutorial](./tutorial.mdx) for an example of this flow. The concept pages in the sidebar to the left go into greater detail of more complex configurations.
+First, check if the API you want to use has an available connector in the catalog. If you find it there, you can use it as is. If you need to update an existing connector, see the guide for updates. 
 
-## Connector vs. configured source vs. connection
+Generally, you can build a connector with the Connector Builder if you want to connect to an HTTP API that returns a collection of records as JSON and has fixed endpoints. For more detailed information on requirements, refer to the [compatibility guide](./connector-builder-compatibility.md). 
 
-When building a connector, it's important to differentiate between the connector, the configured source based on a connector, and the connection:
+## Getting started
 
-The **connector** defines the functionality of how to access an API or a database, for example protocol, URL paths to access, the way requests need to be structured, and how to extract records from responses.
+The step-by-step process for using the connector builder is as follows:
```

Comment:
```suggestion
The high-level process for using the connector builder is as follows:
```

Maybe high-level instead of step-by-step since this is more of an overview of overall process?